index.js

```diff
@@ -329,14 +330,16 @@ FacebookInsightStream.prototype.handleError = function ( error, retry ) {
     }
 }
 
-// predicate-based error filter 
+// predicate-based error filter
 function SkippedError ( error ) {
     return error.skip === true;
 }
 
 function errorHandler ( options, body )  {
     if ( body.error ) {
-        body.error.skip = options.ignoreMissing && body.error.code === MISSING_ERROR_CODE
+        body.error.skip = options.ignoreMissing
+        && (body.error.code === MISSING_ERROR_CODE)
+        || body.error.code === NOT_SUPPORTED_CODE
```

Comment:
Nope. The first two are in order to skip missing items so only if the `ignoreMissing` is `true` and the `code` is the same as `MISSING_ERROR_CODE` we will want to skip it. The new addition is separated from that.